package.json

```diff
@@ -16,7 +16,7 @@
   ],
   "scripts": {
     "linter:fix": "prettier --write '{,!(node_modules|.git|storybook-static)/**/}*.{js,json,jsx,md}'",
-    "linter:check": "eslint src/**/* && eslint stories/*.jsx",
+    "linter:check": "eslint 'src/**/*.{tsx, jsx, ts, js}' && eslint stories/*.jsx",
```

Comment:
```suggestion
    "linter:check": "eslint 'src/**/*' && eslint stories/*.jsx",
```

Reply:
this one will keep the error, no ? as it will not detect the files `Col`, `Row`, under Grid 🤔

Reply:
I tried it and it works locally, but it picks up an extra issue with the index file you should look into. 

I think the difference is this includes quotes and passes the handling to eslint, the other one uses bash to expand and includes folders instead of files